src/platform/posix.c

```diff
@@ -229,8 +232,8 @@ bool platform_lock_single_instance()
 	char pidFilePath[MAX_PATH];
 	char separator = platform_get_path_separator();
 
-	strncpy(pidFilePath, _userDataDirectoryPath, MAX_PATH);
-	strncat(pidFilePath, &separator, 1);
+	strncpy(pidFilePath, _userDataDirectoryPath, MAX_PATH - 1);
+	strncat(pidFilePath, &separator, MAX_PATH - strnlen(pidFilePath, MAX_PATH) - 1);
```

Comment:
The third parameter is maximum amount of characters to copy, so yes, remaining space. That's a common mistake.